readme.md

```diff
@@ -164,6 +164,8 @@ const useSoundStore = create((set, get) => ({
 
 Sometimes you need to access state in a non-reactive way or act upon the store. For these cases, the resulting hook has utility functions attached to its prototype.
 
+:warning: Avoid using this technique to add state in [React Server Components](https://github.com/reactjs/rfcs/blob/main/text/0188-server-components.md). It is an anti-pattern that can cause strange hard-to-solve bugs and unintended leakage of a user's private data to other users. Read [#2200](https://github.com/pmndrs/zustand/discussions/2200) for more info.
```

Comment:
It looks okay to me. It might be a bit too strong, but it might be fine. Maybe change the wording a bit, but I would like to see what others think. I'm not very sure how people misunderstand.

Reply:
Here's a softer version. I also mention Next.js in it.

> ⚠️ This technique is not recommended for adding state in React Server Components (typically in Next.js 13 and above). It can lead to unexpected bugs and privacy issues for your users. For more details, see #2200.

Does this sound better? @dai-shi

Reply:
Yeah, that sounds better!

Reply:
@dai-shi done. You may **Squash and Merge** :)